crates/solana-gateway-task-processor/src/component.rs

```diff
@@ -403,7 +407,7 @@ async fn execute_task(
             )
             .await?;
 
-            send_transaction(solana_rpc_client, keypair, ix).await?;
+            send_gateway_tx(solana_rpc_client, keypair, ix).await?;
```

Comment:
I guess the only gateway errors that could happen in these cases are from the message validation call, right? Taking a closer look at it, it seems we could continue in case the destination contract address is invalid. Is that correct? Does it make sense? I'm also wondering if we really need to use `send_gateway_tx` here. Thoughts?

Reply:
Thanks for the explanation! Makes sense.